var/spack/repos/builtin/packages/nvptx-tools/package.py

```diff
@@ -0,0 +1,49 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class NvptxTools(AutotoolsPackage):
+    """nvptx-tools: A collection of tools for use with nvptx-none GCC
+    toolchains. These tools are necessary when building a version
+    of GCC that enables offloading of OpenMP/OpenACC code to NVIDIA
+    GPUs."""
+
+    homepage = "https://github.com/MentorEmbedded/nvptx-tools"
+    git      = "https://github.com/MentorEmbedded/nvptx-tools"
+
+    version('master', branch='master')
```

Comment:
I'd prefer to see date based versions tied to a commit hash https://spack.readthedocs.io/en/latest/packaging_guide.html#git

Reply:
Fixed, thanks.